### ISSUE TITLE
"send" transaction fails, "out of gas"

### ISSUE THREAD

**Issue by auryn-macmillan** [TomAFrench/colony-safe-app#17]
Sending a token to an external address fails.

https://etherscan.io/tx/0xf27c7e2414314a81a701408aae54fe3df259ce52fa5052b279979eb7cc742c01
>  Although one or more Error Occurred [Out of gas] Contract Execution Completed

For reference sake, a 1txPayment transaction should be about 750,000 gas. This transaction only allocated 130,000 gas by default.

**Comment by auryn-macmillan** [comment]
I tried with 800,000 gas and it still failed, but it only used 300,000 gas.
https://etherscan.io/tx/0xa9358959880bf2dee1e1b2aa20c186483778d1a79505c0cad6cb355ce4040f0e

**Comment by TomAFrench** [comment]
> For reference sake, a 1txPayment transaction should be about 750,000 gas. This transaction only allocated 130,000 gas by default.

I have no control over what the suggested gas limit of a transaction will be but this could be a good addition to the Gnosis sdk if it's not working at the minute.

**Comment by auryn-macmillan** [comment]
My guess is that the safe SDK estimated enough gas to get to the point where the transaction failed. Which is kind of ironic.

**Comment by auryn-macmillan** [comment]
@TomAFrench I don't think this is an issue with the Gnosis Safe underestimating the gas. As I mentioned before, the transaction fails even if you provide it with more gas. See this transaction: https://etherscan.io/tx/0xa9358959880bf2dee1e1b2aa20c186483778d1a79505c0cad6cb355ce4040f0e

**Comment by auryn-macmillan** [comment]
Here is some more information on this.
![image](https://user-images.githubusercontent.com/8453294/96119595-3709e280-0ebb-11eb-96a9-f007a804dbb6.png)

For reference, this is [the transaction](https://discordapp.com/channels/562263648173555742/611571926439165972/766006581778186240).

Apparently we're using the wrong skill ID, which probably explains why the transaction is failing, regardless of how much gas is provided.

We should set the skill ID to `0`.
